topics/assembly/tutorials/vgp_workflow_training/tutorial.md

```diff
@@ -58,102 +58,179 @@ GWS facilitates analysis repeatability, while minimizing the number of manual st
 
+1. A set of PacBio {HiFi} reads in `fasta` format
+2. A set of Illumina {Hi-C} reads in `fastqsanger.gz` format
 
-> <details-title>Working with your own data</details-title>
->
-> If working on a genome other than the example yeast genome, you can upload the VGP data from the [VGP/Genome Ark AWS S3 bucket](https://genomeark.s3.amazonaws.com/index.html) as follows:
->
-> > <hands-on-title>Import data from GenomeArk</hands-on-title>
-> >
-> > 1. Open the file {% icon galaxy-upload %} __upload__ menu
-> > 2. Click on **Choose remote files** tab
-> > 3. Click on the **Genome Ark** button and then click on **species**
-> {: .hands_on}
->
-> You can find the VGP data following this path: `/species/${Genus}_${species}/${specimen_code}/genomic_data`. Inside a given datatype directory (*e.g.* `pacbio`), select all the relevant files individually until all the desired files are highlighted and click the <kbd>Ok</kbd> button. Note that there may be multiple pages of files listed. Also note that you may not want every file listed.
->
-> {% snippet faqs/galaxy/collections_build_list.md %}
->
-{: .details}
+## Uploading `fasta` datasets from Zenodo
+
+The following two steps demonstrate how to upload three PacBio {HiFi} datasets into you Galaxy history.
+
+### **Step 1**: Copy the following URLs into clipboard
+(you can do this by clicking on {% icon copy %} button in the right upper corner of the box below. It will appear if you mouse over the box.)
+
+```
+https://zenodo.org/record/6098306/files/HiFi_synthetic_50x_01.fasta
+https://zenodo.org/record/6098306/files/HiFi_synthetic_50x_02.fasta
+https://zenodo.org/record/6098306/files/HiFi_synthetic_50x_03.fasta
+```
+
+### **Step 2**: Upload datasets into Galaxy
+
+These datasets are in `fasta` format. Upload by following the steps shown in the figure below.
+
+------
+
+![Figure 2: Uploading fasta files in Galaxy]({{site.baseurl}}/faqs/galaxy/images/upload_fasta_via_url.png "Here we upload three fasta files. Compressed (.gz or .bz2) datasets are uploaded in exactly the same fashion by selecting an appropriate datatype (fasta.gz or fasta.bz2)")
+
+------
+
+
+## Uploading `fastqsanger.gz` datasets from Zenodo
+
+Illumina {Hi-C} data is uploaded in essentially the same way as shown in the following two steps.
+
+> <warning-title>DANGER: Make sure you choose correct format!</warning-title>
+> When selecting datatype in "**Type (set all)**" drop-down, make sure you select `fastaqsanger` or `fastqsanger.gz` BUT NOT `fastqcssanger` or anything else!
+{: .warning}
+
+### **Step 1**: Copy the following URLs into clipboard
+(you can do this by clicking on {% icon copy %} button in the right upper corner of the box below. It will appear if you mouse over the box.)
+
+```
+https://zenodo.org/record/5550653/files/SRR7126301_1.fastq.gz
+https://zenodo.org/record/5550653/files/SRR7126301_2.fastq.gz
+```
```

Comment:
This seems like a hands-on step, should it be in a hands-on box to help learners recognise it's a step they need to execute and not just read about?